roles/kubernetes/node/defaults/main.yml

```diff
@@ -20,9 +20,6 @@ kubelet_kubelet_cgroups: "/{{ kube_service_cgroups }}/kubelet.service"
 kubelet_runtime_cgroups_cgroupfs: "/system.slice/{{ container_manager }}.service"
 kubelet_kubelet_cgroups_cgroupfs: "/system.slice/kubelet.service"
 
-### fail with swap on (default true)
-kubelet_fail_swap_on: true
```

Comment:
Hope this is fine. As far as I could see `kubespray-defaults` is imported always before `kubernetes/node` in the playbooks.